istio-2-mixin/dashboards.libsonnet

```diff
@@ -0,0 +1,164 @@
+              panels.clientWorkloadGRPCResponses { gridPos+: { w: 16 } },
+              panels.clientWorkloadTCPRequestThroughput { gridPos+: { w: 12 } },
+              panels.clientWorkloadTCPResponseThroughput { gridPos+: { w: 12 } },
+              g.panel.row.new('Server details'),
+              panels.serverWorkloadHTTPGRPCRequests,
+              panels.serverWorkloadHTTPGRPCRequestDelay,
+              panels.serverWorkloadHTTPGRPCRequestThroughput,
+              panels.serverWorkloadHTTPGRPCResponseThroughput,
+              panels.serverWorkloadHTTPResponseOverview { gridPos+: { w: 8 } },
+              panels.serverWorkloadHTTPResponses { gridPos+: { w: 16 } },
+              panels.serverWorkloadGRPCResponseOverview { gridPos+: { w: 8 } },
+              panels.serverWorkloadGRPCResponses { gridPos+: { w: 16 } },
+              panels.serverWorkloadTCPRequestThroughput { gridPos+: { w: 12 } },
+              panels.serverWorkloadTCPResponseThroughput { gridPos+: { w: 12 } },
+            ], 12, 6
+          )
+        )
+        // hide link to self
+        + root.applyCommon(vars.workloadOverviewVariables, uid + '-workloads-overview', tags, links { backToWorkloadsOverview+:: {} }, annotations, timezone, refresh, period),
+    }
+    +
+    if this.config.enableLokiLogs then
+      {
+        logs:
+          logslib.new(
+            prefix + 'Istio logs',
+            datasourceName=this.grafana.variables.datasources.loki.name,
+            datasourceRegex=this.grafana.variables.datasources.loki.regex,
+            filterSelector=this.config.filteringSelector,
+            labels=this.config.groupLabels + this.config.instanceLabels + this.config.extraLogLabels,
+            formatParser=null,
+            showLogsVolume=this.config.showLogsVolume,
+            logsVolumeGroupBy=this.config.logsVolumeGroupBy,
+          )
+          {
+            dashboards+:
+              {
+                logs+:
+                  // reference to self, already generated variables, to keep them, but apply other common data in applyCommon
+                  root.applyCommon(super.logs.templating.list, uid=uid + '-logs', tags=tags, links=links, annotations=annotations, timezone=timezone, refresh=refresh, period=period),
+              },
+            panels+:
+              {
+                // modify log panel
+                logs+:
+                  g.panel.logs.options.withEnableLogDetails(true)
+                  + g.panel.logs.options.withShowTime(false)
+                  + g.panel.logs.options.withWrapLogMessage(false),
+              },
+            variables+: {
+              // add prometheus datasource for annotations processing
+              toArray+: [
+                this.grafana.variables.datasources.prometheus { hide: 2 },
+              ],
+            },
+          }.dashboards.logs,
+      }
+    else {},
+  //Apply common options(uids, tags, annotations etc..) to all dashboards above
```

Comment:
Are annotations supposed to be added here too? Such as:
  "annotations": {
    "list": [
      {
        "builtIn": 1,
        "datasource": {
          "type": "grafana",
          "uid": "-- Grafana --"
        },
        "enable": true,
        "hide": true,
        "iconColor": "rgba(0, 211, 255, 1)",
        "name": "Annotations & Alerts",
        "type": "dashboard"
      }
    ]
  },